staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -297,6 +297,22 @@ func (o *CopyOptions) copyFromPod(src, dest fileSpec) error {
 		return errFileCannotBeEmpty
 	}
 
+	var (
+		prefix string
+		Command []string
+	)
+	if strings.HasPrefix(src.File, "."){
+		prefix = getPrefix(src.File)
+		prefix = path.Clean(prefix)
+		// remove extraneous path shortcuts - these could occur if a path contained extra "../"
+		// and attempted to navigate beyond "/" in a remote filesystem
+		prefix = stripPathShortcuts(prefix)
+		Command = []string{"tar", "cf", "-", prefix}
+	} else {
+		prefix = src.File
+		Command = []string{"tar", "cPf", "-", src.File}
```

Comment:
I think this introduces a vulnerability.

There's CVE history for `kubectl cp`. I don't think we want to suppress warnings here.